ci/taos/webapp/pr-handler-process.php

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+<meta http-equiv="Content-Type" content="text/html; charset=utf-8">
+<meta name="viewport" content="width=device-width, user-scalable=no">
+<title>PR Processor</title>
+</head>
+<body>
+<?php
+
+// Author: Geunsik Lim
+// Title: A processor to handle a request of PRs
+//
+
+include('webapp_config.php');
+
+// change last character to 0.
+// For /example, convert 201805051635 to 201805051630
+date_default_timezone_set("Asia/Seoul");
+
+
+// This line is used for just debugging.
+//$t=time();
+//$curr_time = date("YmdHi",$t);
+//$curr_time = substr_replace($curr_time, "", -1)."0";
+
+$store_name = $_POST['store'];
+$event_msg = $_POST['message']; 
+$time_start_year = $_POST['start_year'];
+$time_start_month = $_POST['start_month'];
+$time_start_day = $_POST['start_day'];
+$time_start_hour = $_POST['start_hour'];
+$time_start_minute = $_POST['start_minute'];
+$event_password = $_POST['password'];
+
+if(empty($event_msg)) {
+    echo "<script>window.alert('입력내용을 작성하여 주세요.');</script>";
+    echo "<script>window.location='./event_upload.php';</script>";
+}
+else {
+// get ip address of user
+function get_client_ip()
```

Comment:
Indentation is strange. Is this intentional?

Reply:
Yes. And this PR is a draft version.